tests/run/t6476.scala

```diff
@@ -0,0 +1,23 @@
+object Test {
+  def main(args: Array[String]): Unit = {
+    val person = "Alice"
+    println(s"\"Hello\", $person")
+    println(s"""\"Hello\", $person""")
+
+    println(f"\"Hello\", $person")
```

Comment:
It looks like this one was wrongly interpreted as `\"` instead of `"`. The check file differs with the one in Scala 2.

Reply:
I filed https://github.com/lampepfl/dotty/issues/11750 about that